strawberry/federation/field.py

```diff
@@ -87,13 +94,16 @@ def field(
     name: Optional[str] = None,
     is_subscription: bool = False,
     description: Optional[str] = None,
+    authenticated: bool = False,
+    external: bool = False,
+    inaccessible: bool = False,
+    policy: Optional[List[List[Federation__Policy]]] = None,
```

Comment:
I think we should change this to 
```suggestion
    policy: Optional[List[List[str]]] = None,
```

`Federation__Policy` is a scalar, and at the moment the typing for them is broken 😊

Reply:
done. Can you offer any suggestion on how to fix the failing benchmark tests? i can't repo that error on my local machine.

Reply:
we can ignore that test 😊

I'll review and merge this tonight or tomorrow, thank you so much!